Needed for cloudbase-init to run on non-Windows systems.  Needs to go upstream.

--- cloudbase-init-0.9.8a/cloudbaseinit/osutils/posix.py	2015-03-19 15:42:23.000000000 -0700
+++ /tmp/posix.py.new	2015-03-19 16:05:45.328238028 -0700
@@ -17,7 +17,14 @@
 from cloudbaseinit.osutils import base
 
 
-class PosixUtil(base.BaseOSUtils):
+class PosixUtils(base.BaseOSUtils):
 
     def reboot(self):
         os.system('reboot')
+
+    def get_config_value(self, name, section=None):
+        pass
+
+    def set_config_value(self, name, value, section=None):
+        pass
+
